src/services/tradeApiClient.js

```diff
@@ -2120,6 +2118,22 @@ class TradeApiClient {
     const responseData = await this.doRequest(endpointPath, {}, "POST", 2);
     return responseData;
   }
+
+  /**
+   * Generate user exchange positions report
+   *
+   * @param {InternalTransferPayload} payload Payload to generate position report
+   *
+   * @returns {Promise<boolean>} Result
+   *
+   * @memberof TradeApiClient
+   */
+  async performInternalTransfer(payload) {
+    const endpointPath = `/user/exchanges/${payload.internalIdSrc}/internal_transfer`;
+    delete payload.internalIdSrc;
```

Comment:
no need

Reply:
Miguel mentione sometime ago that we should only send those parameters that are expected so that why I added this. maybe here its okay. removed!

Reply:
But you're not passing payload anyway

Reply:
my bad here, fixed it with new PR.